ratarmount.py

```diff
@@ -18,7 +18,12 @@
 import urllib.parse
 import zipfile
 from typing import Any, Callable, Dict, Iterable, IO, List, Optional, Tuple, Union
-import fuse
+
+try:
```

Comment:
Why use try-catch? It won't work anyway when fuse cannot be imported.

Reply:
On my sytsem  import fuse is not recognized, i need to import fusepy